packages/swap/src/routes/swap.ts

```diff
@@ -277,6 +290,9 @@ router.get(
       failedAt: failedSwap?.failedAt,
       failedBlockIndex: failedSwap?.failedBlockIndex ?? undefined,
       depositChannelAffiliateBrokers: affiliateBrokers,
+      depositChannelMaxBoostFeeBps: swapDepositChannel?.maxBoostFeeBps,
+      effectiveBoostFeeBps: swap?.effectiveBoostFeeBps,
+      boosted,
```

Comment:
what do you think about setting the `effectiveBoostFeeBps` to 0 instead of exposing another property?

Reply:
so `effectiveBoostFeeBps` would be undefined until the deposit is prewitnessed. after that, it will be either the effective boost fee or 0 if the deposit was not boosted

Reply:
Yeah, i guess that would simplify things. I actually exposed the `boosted` property for convenience for integrators. Its not necessary though